### DISCUSSION TITLE
Replaced OPI5 charger from 3A to 5A

### DISCUSSION THREAD

**Discussion by Bahazzie** [Joshua-Riek/ubuntu-rockchip#1059] [Q&A]
Do you think it could help with the inexplicable desktop logoffs (probably due to crashes under the hood) I was having? There was also some USB gear plugged in (mouse, ssd, keyboard). Anyway based on Joshua's questions to others (about power being 3A in stead of 5) I decided to replace my 3A charger, let's see how it goes...Bas

**Comment by Joshua-Riek** [comment]
Try 5v 4a and it will work. 5v 3a is not enough power.

**Comment by Bahazzie** [comment]
Yes, but I use 5V/5A charger now, also fit for rpi5. But alas, I had one spontaneous logout with the new 5A charger too. Using the RPI headless with RealVNC server (hacked that RealVNC together). Is there a way I can analyze the cause? Can I collect crash logs somehow? How?

Bas

**Comment by Bahazzie** [comment]
Possibly related to Chrome/Chromium in combination with Xorg (I disabled Wayland for VNC). Chromium crashes Xorg sometimes. Since I  switched to Firefox I haven't had a single Xorg logout.
